tests-ui/tests/store/modelToNodeStore.test.ts

```diff
@@ -0,0 +1,245 @@
+import { createPinia, setActivePinia } from 'pinia'
+import { beforeEach, describe, expect, it } from 'vitest'
+
+import {
+  ModelNodeProvider,
+  useModelToNodeStore
+} from '@/stores/modelToNodeStore'
+import { type ComfyNodeDefImpl, useNodeDefStore } from '@/stores/nodeDefStore'
+
+describe('useModelToNodeStore', () => {
+  let store: ReturnType<typeof useModelToNodeStore>
+  let nodeDefStore: ReturnType<typeof useNodeDefStore>
+
+  // Create minimal mock for testing - only includes 'name' field since that's
+  // the only property ModelNodeProvider constructor uses and tests verify
+  const createMockNodeDef = (name: string): ComfyNodeDefImpl => {
+    return { name } as ComfyNodeDefImpl
+  }
+
+  beforeEach(() => {
+    setActivePinia(createPinia())
+    store = useModelToNodeStore()
+    nodeDefStore = useNodeDefStore()
```

Comment:
Also, why is one store and the other nodeDefStore? Feels like playing favorites 😛